### ISSUE TITLE
Refactor|Rename not editing correctly

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#2319]
Given the declaration:

```
  Dim oCell As Range
    For Each oCell In DestinationSheet.UsedRange
```
Refactor|Rename to change `oCell` to `Rng` yields: 

```
  Dim Rng As Range
    For Each Rngn DestinationSheet.UsedRange
```

Refactor|Rename to change `oCell` to `MyCell` yields:

```
  Dim MyCell As Range
    For Each MyCelll In DestinationSheet.UsedRange
```
(Note the extra `l` in the usage)

This is very consistent and reproducible in Excel 2010, RD 2.0.8 on _this_ variable and usage. I used RD to rename quite a few other variables and it worked just fine.

Attaching a sample .XLTM. Look in `PasteInProtectedWorksheet.UnProtectPasteToSheet`.
[Screening Template - RD Sample.zip](https://github.com/rubberduck-vba/Rubberduck/files/536393/Screening.Template.-.RD.Sample.zip)

**Comment by retailcoder** [comment]
Fixed in dcd672f871f800b3bb9b660c593d58cb1459cafd (released with 2.0.9) :smiley:

**Comment by retailcoder** [comment]
Duplicate of #2276
